tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@grafana/tsconfig/base.json",
   "include": ["src", "index.js"],
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
I'm somewhat unclear on this change tbh, I needed it locally but Idk if that means there's something wrong with my setup?

Reply:
if it's not in the @grafana/tsconfig/base.json then it's fine to add this here